docs/en/writing-running-appium/android/android-appbundle.md

```diff
@@ -0,0 +1,51 @@
+## How To Test Android App Bundle
+
+Google has released [Android App Bundle](https://developer.android.com/platform/technology/app-bundle/) feature.
+`.aab` file is generated by the feature. We must upload the file to Google Play Store.
+
+We can get distributed apk files from the `.aab` file via [bundletool](https://github.com/google/bundletool). Using the generated files, we can test against release module. After [1](https://github.com/appium/appium-adb/pull/367) and [2](https://github.com/appium/appium-base-driver/pull/271), you can run Appium tests against `.apks` file. The commits are available over Appium 1.9.2.
+
+## How to run tests
+
+1. Export `bundletool.jar` in your path
+    - Appium search the `bundletool.jar` in your local. Make sure you can find the path with `which 'bundletool.jar'`. If you can't find it, please set the path correctly.
+2. Generate `.apks` from `.aab`
+    - You must sign correctly when you generate `.apks` from `.aab`. This step requires signing data.
+        ```bash
+        $ java -jar apks/bundletool-all-0.6.0.jar build-apks \
+          --bundle apks/release/release/app.aab \ # A generated aab file
+          --output apks/AppBundleSample.apks \    # A apks file you'd like to out put to
+          --ks apks/sign \                        # Signing keystore
+          --ks-key-alias key0 \                   # Alias of the keytstore
+          --ks-pass pass:kazucocoa \              # Password of the keystore
+          --overwrite                             # Everytime, you can override the apks
+        ```
+3. Set the `.apks` in your `app` capability.
+    ```ruby
+    desired_capability = caps: {
+        platformName: :android,
+        automationName: 'uiautomator2',
+        platformVersion: '8.1',
+        deviceName: 'Android Emulator',
+        app: "path/to/your.apks",   # This line is important
+        fullReset: true,
+        ...
+    }
+
+    core = ::Appium::Core.for(desired_capability)
+    driver = core.start_driver
+    ```
+
+## Tips
+
+### Test with differnet languages
+
+Set `fullReset: true` if you would like to test against different languages resources in every creating session.
+
+Appium installs minimal resources following appbundle feature. For example, if a device environment is English, Appium installs only `en` resource. The installed apk has no Japanese resource.
+
+Appium decides which resource must install in the installation phase. We should specify `fullReset: true` to force re-installation call.
```

Comment:
In order to force re-installation with a different set of language resources, specify `fullReset: true`